src/client/sandbox/event/focus-blur.js

```diff
@@ -115,8 +115,8 @@ export default class FocusBlurSandbox extends SandboxBase {
 
 
     _raiseEvent (el, type, callback, withoutHandlers, isAsync, forMouseEvent, preventScrolling) {
-        // NOTE: We cannot use Promise from the es6-promise library because the 'resolve' method is called
-        // from setTimeout(1) in IE9 and IE10.
+        // NOTE: We cannot use Promise because 'resolve' will be called async, but we need to resolve
+        // immediately in IE9 and IE10.
 
```

Comment:
We've decided to not refactor this code right now, because we shouldn't rely on how promise lib executes the code. We should refactor the whole code so we will explicitly use Promise based function there we expect async behavior, and don't use it there we expect sync (see #179)